### PR TITLE
Add ability to process pull requests

### PR DESCRIPTION
**Pull request by jleach**
First feature of several to codify cultural policies surrounding what a good PR should look like. This PR specifically deals with a PR length (lines added + lines removed). It monitors a configurable property called `maxLinesChanged` and if a PR is over that limit it will add a comment indicating that smaller PRs are preferred. Configurable properties are stored in a file that is in tern stored in each repo.

### CODE REVIEW

File: src/libs/pullrequest.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+ * The length of a PR is determined by adding the lines deleted and added
+ * @param {Context} context The event context context
+ * @param {RepoMountieConfig} config The repo config file
+ * @returns True if the length is valid, False otherwise
+ */
+export const isValidPullRequestLength = (context: Context, config: RepoMountieConfig): boolean => {
+  const linesChanged =
+    context.payload.pull_request.additions + context.payload.pull_request.deletions;
+
+  if (linesChanged < config.pullRequest.maxLinesChanged) {
```
Comment (ShellyXueHan) [inline src/libs/pullrequest.ts]: max line means `linesChanged <= config.pullRequest.maxLinesChanged`?

File: __tests__/pullrequest.test.ts [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+  test('A PR with the same number of changes is rejected', () => {
+    context.payload.pull_request.additions = 10;
+    const myConfig = { ...repoMountieConfig };
+    myConfig.pullRequest.maxLinesChanged = 10;
+
+    expect(isValidPullRequestLength(context, myConfig)).toBeFalsy();
+  });
+
+  test('Blarb', async () => {
```
Comment (ShellyXueHan) [inline __tests__/pullrequest.test.ts]: Any better description?

Comment (jleach) [reply __tests__/pullrequest.test.ts]: Fixed

File: src/libs/pullrequest.ts (hunk 1 above)
Comment (jleach) [reply src/libs/pullrequest.ts]: Fixed.

**Pull request merged**